### DISCUSSION TITLE
Non-Crowdin localization management platform?

### DISCUSSION THREAD

**Discussion by angelovdev** [amannn/next-intl#536] [Q&A]
I know next-intl partnered with Crowdin but I am wondering if there is integration with any other (potentially open source, self-hosted) localization management platform or plans to have such integration(s)?

**Comment by amannn** [comment]
`next-intl` absolutely works with other localization management platforms as well as without one. The only requirement that `next-intl` has, is that messages should be available as a nested JSON file. Did you run into a particular problem?
